applications/DEM_application/custom_elements/spheric_particle.cpp

```diff
@@ -1434,9 +1434,9 @@ void SphericParticle::ComputeAdditionalForces(array_1d<double, 3>& externally_ap
         const double vel_magnitude = DEM_MODULUS_3(vel);
         if (vel_magnitude != 0.0){
             const array_1d<double, 3> unitary_vel =  vel/vel_magnitude;
-            const double inlet_damping_coefficient = 1e4;
-            const array_1d<double, 3> damping_force = - inlet_damping_coefficient * GetMass() * vel;
-            const array_1d<double, 3> counter_force  = - 50.0 * gravity_force_magnitude * unitary_vel;
+            const double inlet_damping_coefficient = 1e3;
+            const array_1d<double, 3> damping_force = - inlet_damping_coefficient * GetMass() * vel_magnitude  * vel_magnitude * unitary_vel;
+            const array_1d<double, 3> counter_force  = -25.0 * gravity_force_magnitude * unitary_vel;
```

Comment:
why is this force so big? 25 times the gravity? I would have used like 2 or 3...

Reply:
Only tested with 50 and 25. 25 looks fine.